app/models/alchemy/element.rb

```diff
@@ -7,7 +7,7 @@
 #  id                :integer          not null, primary key
 #  name              :string
 #  position          :integer
-#  page_id           :integer          not null
+#  page_version_id   :integer          not null
```

Comment:
The commit message here misses the word `version` in the body

Reply:
Done